src/robusta/core/discovery/discovery.py

```diff
@@ -409,6 +409,12 @@ def discover_stats() -> ClusterStats:
         except Exception:
             logging.error("Failed to count jobs", exc_info=True)
 
+        k8s_version: str = None
+        try:
+            k8s_version = client.VersionApi().get_code().git_version
```

Comment:
can this change without restarting runner?
It might be something we only need to check once instead of calling it every five minutes

Reply:
it is possible to update Kubernetes version without restart